src/xrpld/app/misc/WasmHostFuncWrapper.cpp

```diff
@@ -801,6 +837,33 @@ accountKeylet_wrap(
         runtime, params, results, hf->accountKeylet(*acc), index);
 }
 
+wasm_trap_t*
+ammKeylet_wrap(void* env, wasm_val_vec_t const* params, wasm_val_vec_t* results)
+{
+    auto* hf = reinterpret_cast<HostFunctions*>(env);
+    auto const* runtime = reinterpret_cast<InstanceWrapper const*>(hf->getRT());
+    int index = 0;
+
+    auto const issue1 = getDataAsset(runtime, params, index);
```

Comment:
asset1

Reply:
getDataAsset  is it return asset or issue?

Reply:
It returns an asset, but that's what the keylet text shows.